pandas/core/internals/managers.py

```diff
@@ -1675,8 +1677,11 @@ def create_block_manager_from_arrays(
     assert isinstance(axes, list)
     assert all(isinstance(x, Index) for x in axes)
 
+    # ensure we dont have any PandasArrays when we call get_block_type
```

Comment:
what is this about?

Reply:
ATM if we pass a PandasArray to Series we end up unwrapping it to an ndarray but it still gets put into an ExtensionBlock because we dont unwrap it early enough